sdks/python/apache_beam/runners/dataflow/dataflow_runner_test.py

```diff
@@ -279,8 +282,7 @@ def process(self):
                      {'type': 'INTEGER', 'namespace': nspace+'SpecialDoFn',
                       'value': 42, 'key': 'dofn_value'}]
     expected_data = sorted(expected_data, key=lambda x: x['namespace']+x['key'])
```

Comment:
Is this invocation of sorted still necessary, since you've removed the other one?

Reply:
ah good catch, not required. PTAL